src/prng/isaac64.rs

```diff
@@ -317,26 +310,28 @@ fn mix(a: &mut w64, b: &mut w64, c: &mut w64, d: &mut w64,
 
 impl SeedFromRng for Isaac64Rng {
     fn from_rng<R: Rng>(mut other: R) -> Result<Self, Error> {
-        let mut key = [w(0); RAND_SIZE];
+        let mut seed = [w(0); RAND_SIZE];
         unsafe {
-            let ptr = key.as_mut_ptr() as *mut u8;
+            let ptr = seed.as_mut_ptr() as *mut u8;
```

Comment:
and here

Reply:
All four are now share the same code :smile: